packages/sdk/base/src/async.ts

```diff
@@ -139,3 +139,29 @@ export async function concurrentValuesMap<IN extends any, OUT extends any>(
     return output
   }, {})
 }
+
+export const timeout = <T extends any[], U>(
```

Comment:
Maybe add some comment here on what the timeout behavior is. Right now in here it's not obvious that it would throw an error, but that's what the test seems to indicate

Reply:
will do